### DISCUSSION TITLE
Detected a Cloudflare version 2 Captcha challenge

### DISCUSSION THREAD

**Discussion by Bruchogun** [jorgeajimenezl/animeflv-api#12] [Q&A]
looks like the API is not working due to inability to surpass the captcha

`cloudscraper.exceptions.CloudflareChallengeError: Detected a Cloudflare version 2 Captcha challenge, This feature is not available in the opensource (free) version.
PS C:\Users\mauro\Lab 7\Lab 7.srcs\sources_1\new> `

**Comment by jorgeajimenezl** [comment]
Hi @Bruchogun, see #4
